sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -79,10 +132,13 @@ struct joint_matrix {
 }
 ```
 
-IMPORTANT: Matrix layout defaulting to `layout::dynamic` applies only to matrix with `use::accumulator`
+IMPORTANT: Matrix layout defaulting to `layout::dynamic` applies only
+to matrix with `use::accumulator`
 
-#### Use
-Specifying the usage of the matrix: matrix left (A), matrix right (B) or accumulator +(C)+ is required by backend implementations to reason about the layout of the matrix in registers.
+==== Use
+Specifying the usage of the matrix: matrix left (A), matrix right (B)
```

Comment:
`matrix left` and `matrix right` are not defined (or equivalently A / B aren't defined).